.mergify.yml

```diff
@@ -10,12 +10,20 @@ pull_request_rules:
   - name: automatic merge
     conditions:
       - label!=DNM
-      - '#approved-reviews-by>=2'
-      - 'status-success=continuous-integration/travis-ci/pr'
+      - "#approved-reviews-by>=2"
+      - "status-success=continuous-integration/travis-ci/pr"
```

Comment:
Why change the quoting?

Reply:
done by the editor, mergify rules are verified its correct

Reply:
Ok.

**Nit:** It is better to make these changes as a separate commit, or mention is in the commit message as not being related to the functional change. What should be done when someone else uses a different editor/settings and the quoting gets replaced again?

Reply:
good  question removed the change